### PR TITLE
Server logs improvements #48

### PR DESCRIPTION
**Pull request by amper43**
- add analytics logs to server stdout in production mode
- small fixes

### CODE REVIEW

File: server/src/services/analytics_service.ts [hunk 1]
```diff
@@ -91,12 +92,12 @@ export class AnalyticsService {
         console.error('Can`t run analytics process: %s', error);
         return;
       }
-      console.log('Ok, pid: %s', cp.pid);
+      console.log('Alanytics creating successful, pid: %s', cp.pid);
```
Comment (rozetko) [inline server/src/services/analytics_service.ts]: alanytics, lol

File: server/src/services/analytics_service.ts [hunk 2]
```diff
@@ -136,24 +137,21 @@ export class AnalyticsService {
     return new Promise<childProcess.ChildProcess>((resolve, reject) => {
       var resolved = false;
 
-      cp.stdout.on('data', () => {
-        if(resolved) {
-          return;
-        } else {
+      cp.stdout.on('data', (data) => {
+        console.log(data);
+        if(!resolved) {
```
Comment (rozetko) [inline server/src/services/analytics_service.ts]: Do you think `if(resolved) { return; }` not needed?

File: server/src/services/analytics_service.ts [hunk 3]
```diff
@@ -136,24 +137,21 @@ export class AnalyticsService {
...
         }
         resolve(cp);
       });
 
       cp.stderr.on('data', function(data) {
-        if(resolved) {
-          return;
-        } else {
+        console.error(data);
+        if(!resolved) {
```
Comment (rozetko) [inline server/src/services/analytics_service.ts]: The same thing here

File: server/src/services/analytics_service.ts [hunk 4]
```diff
@@ -47,6 +47,7 @@ export class AnalyticsService {
     return new Promise<void>((resolve, reject) => {
       this._requester.send(text, undefined, (err: any) => {
         if(err) {
+          console.trace(`got error while sending ${err}`);
```
Comment (jonyrock-back) [inline server/src/services/analytics_service.ts]: @amper43 can you please explain why `trace` and not `error` is here?

**Pull request merged**